lambdas/src/common/config/config-service.ts

```diff
@@ -76,22 +76,20 @@ export class ConfigService {
     }
 
     public getAuthorizationCodeExpirationEpoch() {
-        // TODO: consider if this should be output in epoch seconds rather than milliseconds
-        // so that it is consistent with the java implementation
-        return Date.now() + this.authorizationCodeTtlInMillis;
+        return Math.floor((Date.now() + this.authorizationCodeTtlInMillis) / 1000);
     }
 
     public getSessionExpirationEpoch() {
         const sessionTtl = this.getConfigEntry(CommonConfigKey.SESSION_TTL);
-        return Date.now() + parseInt(sessionTtl, 10) * 1000;
+        return Math.floor((Date.now() + parseInt(sessionTtl, 10) * 1000) / 1000);
```

Comment:
Could we move the `parseInt` onto the previous line? Then this line would be much more similar to the other ones, only containing the ms -> s calcuations